packages/genesys-spark-components/src/genesys-spark-utils/get-closest-element.ts

```diff
@@ -0,0 +1,23 @@
+//https://inindca.atlassian.net/browse/COMUI-2673
```

Comment:
Thoughts on having a more general element helper file? I feel like this current one will only ever have one method for getting the closest element.